project1/src/main/java/uk/co/cpsd/javaproject1/World.java

```diff
@@ -1,22 +1,37 @@
 package uk.co.cpsd.javaproject1;
 
+import java.io.FileWriter;
+import java.io.IOException;
 import java.util.ArrayList;
+import java.util.HashSet;
 import java.util.List;
+import java.util.Set;
 import java.util.stream.Stream;
 
+
 public class World {
     public final int size = 10;
     private List<Animal> animals;
     private int totalTicks;
-    private final int MAX_GRASS_AGE=25;
+    private final int MAX_GRASS_AGE=100;
     private int[][] grassDeathTime= new int[size][size]; 
+    private List<Integer> goatHistory = new ArrayList<>();
```

Comment:
goatHistory doesn't really tell me enough.  Naming variables is one of the hardest things in software engineering, so I appreciate this is hard.  goatPopulationHistory ?   goatAverageEnergyHistory?  Be more specific